inc/add_site_ui/namespace.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Altis\CMS\Add_Site_UI;
+
+/**
+ * Setup the Add Site UI.
+ */
+function bootstrap() {
+	add_action( 'network_admin_menu', __NAMESPACE__ . '\\register_menu_page' );
+	add_action( 'admin_init', __NAMESPACE__ . '\\add_site_form_handler' );
+}
+
+/**
+ * Register the menu page for the overrided Add New Site page.
+ *
+ * @return void
+ */
+function register_menu_page() {
+	if ( ! is_network_admin() ) {
+		return;
+	}
+	remove_submenu_page( 'sites.php', 'site-new.php' );
```

Comment:
I wonder whether we should hook in similar to the About page so that the URL doesn't change? This would ensure existing links work.

If not, we should at least redirect.